.storybook/main.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     // Components
     '../projects/canopy/src/lib/accordion/docs/guide.stories.mdx',
     '../projects/canopy/src/lib/accordion/docs/accordion.stories.ts',
+    '../projects/canopy/src/lib/carousel/docs/carousel.stories.ts',
```

Comment:
This was actually done on purpose as the use of the Carousel should be discouraged. This was obviously a decision that was taken with @alexcanning before leaving for maternity leave so not sure if anything changed in the meantime!